GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -187,6 +189,8 @@ public RevisionGridControl()
             _gridView.AddColumn(new DateColumnProvider(this));
             _gridView.AddColumn(new CommitIdColumnProvider(this));
             _gridView.AddColumn(_buildServerWatcher.ColumnProvider);
+            _messageColumn = _gridView.Columns[1];
+            _resizableColumns = (from DataGridViewColumn column in _gridView.Columns where column.Resizable == DataGridViewTriState.True select column).ToList();
```

Comment:
You are so right with these suggestions, thank you @drewnoakes.

`Where()` was not available automatically. Usually C# can do the obvious things itself. I don't understand the necessity for the `Cast<>()` function but it's not important to me.

Reply:
`Cast` is required to convert `DataGridViewColumnCollection` to `IEnumerable< DataGridViewColumn>`

Reply:
WinForms predates generics and many of the collection types don't implement `IEnumerable<T>`, only `IEnumerable`. Using `Cast` makes a non-generic enumerable generic. It will throw at runtime if an item in the source enumerable is not of the required type, however.